### DISCUSSION TITLE
Do we have customer view functionality in NexoPoS?

### DISCUSSION THREAD

**Discussion by danielsonfire** [Blair2004/NexoPOS#1357] [General]
I'm interested in using NexoPOS for my business, and I was wondering if the software has customer view functionality? I'm looking for a system that allows my customers to see what they're purchasing and confirm the accuracy of their transactions. Thank you!

**Comment by Blair2004** [comment]
This is coming on v5 as customers will be able to login

**Reply by danielsonfire** [reply]
In the supermarket, I want to display for customer to view the POS price with items. I am not expecting login for customers.
